scopes/webpack/webpack/config/webpack.dev.config.ts

```diff
@@ -103,29 +95,19 @@ export function configFactory(
       // Enable compression
       compress: true,
 
-      // Use 'ws' instead of 'sockjs-node' on server since we're using native
-      // websockets in `webpackHotDevClient`.
-      transportMode: 'ws',
-
       // Enable hot reloading
       hot: true,
 
-      host,
-
       historyApiFallback: {
         disableDotRule: true,
         index: resolveWorkspacePath(publicDirectory),
       },
 
       client: {
-        needClientEntry: false,
         overlay: false,
-        host: clientHost,
-        path: clientPath,
-        port,
       },
```

Comment:
it's bad to put it here, since a low of webpacks extend this, and we could have conflicts when they try to use the same host/port etc